server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -211,11 +222,10 @@ export default class ServiceProviderReferralsController {
       )
     }
 
-    let assignee: AuthUserDetails
     const token = await this.hmppsAuthService.getApiClientToken()
 
     try {
-      assignee = await this.hmppsAuthService.getSPUserByEmailAddress(token, email)
+      await this.hmppsAuthService.getSPUserByEmailAddress(token, email)
```

Comment:
Possibly a silly question, but what's this line doing, now that we're not using the return value?

Reply:
It's to check whether the user exists and to show an error page if not (see the adjacent `catch`).